### ISSUE TITLE
Llama2-Code-Interpreter needs an openai_api_key ?

### ISSUE THREAD

**Issue by itsPreto** [SeungyounShin/Llama2-Code-Interpreter#4]
First off the project looks really well architected. Good job!

I'm excited to try this out but right off the bat I'm not sure if I missed something but it's saying I need an OpenAI API key?

<img width="1728" alt="Screenshot 2023-07-23 at 9 55 37 PM" src="https://github.com/SeungyounShin/Llama2-Code-Interpreter/assets/45348368/e606eea1-27bd-4f97-92e2-b065c64dc454">

**Comment by dillfrescott** [comment]
I am facing this issue too. This project seems sus

**Comment by Jeffjewett27** [comment]
The problem is in BaseCodeInterpreter.py

```python
import openai
from retrying import retry
import logging
from termcolor import colored
# load from key file
with open('./openai_api_key.txt') as f:
    OPENAI_API_KEY = key = f.read()
openai.api_key = OPENAI_API_KEY
``` 

The OpenAI-specific stuff should be moved to GPTCodeInterpreter.py

**Comment by SeungyounShin** [comment]
Sorry for inconvenience I solved the issue at #cd2cb0422674b17b874663b5b78bc904c382ef00
